cmd/internal/server/core.go

```diff
@@ -129,6 +146,14 @@ func (s *Server) shutdown() {
 	// Close any open journals on shutdown to ensure remaining entries are flushed to disk.
 	activity.CloseAll()
 
+	// Flush any outstanding analytics hits
+	ticker := time.NewTicker(analytics.FlushDuration)
+	defer ticker.Stop()
+	go func() {
+		s.analyticsClient.Flush()
```

Comment:
Think it was to prevent Flush from blocking the server process from the rest of its cleanup (in case the network takes too long). I'll add a comment about that.

Reply:
Yeah, as a timer that makes sense.